### ISSUE TITLE
Move subtitles to a different rendering layer?

### ISSUE THREAD

**Issue by newkind** [Stremio/stremio-bugs#1806]
Hey!

So this might be a stretch but I'm using Stremio on my Android TV and I have a setup that is using Hyperion.ng ambilight system alongside with Hyperion Android Grabber which basically grabs whats on the screen and sends to Hyperion.ng to control the ambilight leds. It works awesome but I noticed that the grabber picks also the rendered subtitles and not the video feed only. Please see the below photo. The yellow color below the TV comes from the subtitles, not the video itself.

I was wondering if there would be a possibility of moving the subtitles to a different layer (or something like that) so that the grabber would only grab the video feed and not video combined with the subtitles.

![CleanShot 2024-08-20 at 19 17 29](https://github.com/user-attachments/assets/9351fe26-7599-44ec-ae11-782668a1d3f3)

**Comment by TRtomasz** [comment]
is this still happening after latest update?

**Comment by newkind** [comment]
@TRtomasz yes, its still happening and not only with subtitles but also with all other player UI elements too ie. purple progress bar.
